pkg/build/build.go

```diff
@@ -1086,7 +1086,7 @@ func (ctx *Context) BuildPackage() error {
 	if err := generator.GenerateSBOM(&sbom.Spec{
 		Path:           filepath.Join(ctx.WorkspaceDir, "melange-out", ctx.Configuration.Package.Name),
 		PackageName:    ctx.Configuration.Package.Name,
-		PackageVersion: ctx.Configuration.Package.Version,
+		PackageVersion: fmt.Sprintf("%s-r%d", ctx.Configuration.Package.Version, ctx.Configuration.Package.Epoch),
```

Comment:
the `r` is expected?
besides that lgtm

Reply:
Yes — we want apk versions to look like `1.35.0-r4` (if the epoch is `4`)